SGoopas/Assets/Scripts/Game/Triggers/ButtonInteractTrigger.cs

```diff
@@ -1,23 +1,32 @@
 ﻿using PlayerStates;
-using UnityEngine;
+using UnityEngine;
 
 /*
  * Will cause a button to scale when you press it.
- */
+ */
 public class ButtonInteractTrigger : InteractTrigger {
     private Vector3 originalScale;
-    private bool triggerOn = false;
-
+    private bool triggerOn = false;
+    public AudioClip audioClip;
+    public AudioSource audioSource;

```

Comment:
Can you make it so you don’t have to set this up in the editor? Ideally the buttons will just work, you shouldn’t have to set the sound every time for each button.